README.md

```diff
@@ -2,15 +2,18 @@
 *Bot lists must meet these minimum requirements to be considered a bot list.*
 
 ---
-
-
-1. Your bot list must have a respectable TLD. (.tk does not count)
-2. Your bot list can **NOT** be hosted on glitch.
-3. Your bot list should not display all the listed bots on the front page.
-4. Your bot list must be usable. No broken links/buttons.
-5. Your bot list must have consistent uptime. (Minor downtime for fixing a problem is fine)
-
-
+Your bot list...
+1. must have a respectable TLD.
+   - .tk TLDs are not allowed.
+   - No subdomains used from non-botlists/free hosting (e.g. Glitch.me)
+2. can **NOT** be hosted on glitch.
```

Comment:
See change to master.

Reply:
The PR was made right AFTER the master branch was updated...

Reply:
Please rebase or merge master in to resolve the conflicts.

Reply:
I don't have experience with doing that through github and honestly won't load it on my PC (Desktop) to do this with git.